fxmanifest.lua

```diff
@@ -8,6 +8,7 @@ version '1.0.0'
 shared_scripts {
 	'@ox_lib/init.lua',
 	'@qbx_core/modules/utils.lua',
```

Comment:
Can you yeet utils? It's no longer needed.

Reply:
yeeted